server/src/main/java/org/elasticsearch/search/aggregations/metrics/TDigestExecutionHint.java

```diff
@@ -21,6 +22,14 @@ public enum TDigestExecutionHint implements Writeable {
     DEFAULT(0),        // Use a TDigest that is optimized for performance, with a small penalty in accuracy.
     HIGH_ACCURACY(1);  // Use a TDigest that is optimize for accuracy, at the expense of performance.
 
+    public static final Setting<String> SETTING = Setting.simpleString(
+        "search.aggs.tdigest_execution_hint",
+        "",
```

Comment:
maybe define the default here?

Reply:
Done -  empty string was already mapped to DEFAULT, but made it more explicit.